### DISCUSSION TITLE
Entferne Deutsche Dialekte als unterstütze Sprache

### DISCUSSION THREAD

**Discussion by MrKrisKrisu** [Traewelling/traewelling#2208] [Ideas / Feature Requests]
Ich würde gerne zur Diskussion stellen, ob wir die Deutschen "Dialekte" Bayerisch, Hessisch und Pfälzisch weiter als unterstützte Sprache behalten möchten. Die Übersetzungen werden mittlerweile nicht mehr aktiv gepflegt.

Mir persönlich ist es tatsächlich egal, aber ich möchte gerne das Repo sauber halten - daher würde ich im Zweifel für ein Entfernen sprechen.

![image](https://github.com/Traewelling/traewelling/assets/4103693/8d30ef03-d6ca-44b3-90b2-c9cf44199d83)

See https://translate.codeberg.org/projects/trawelling/traewelling/

**Comment by NyCodeGHG** [comment]
gibt es zahlen wie viele user die tatsächlich benutzen?
das wäre vielleicht gut zu wissen

**Reply by MrKrisKrisu** [reply]
![image](https://github.com/Traewelling/traewelling/assets/4103693/12040789-343d-4d90-b44b-3fcd7397c264)
